platform-sdk/swirlds-fcqueue/src/main/java/com/swirlds/fcqueue/FCQueueStatistics.java

```diff
@@ -20,13 +20,11 @@
 
 import com.swirlds.common.metrics.Metrics;
 import com.swirlds.common.metrics.RunningAverageMetric;
-import com.swirlds.common.system.Platform;
-import com.swirlds.common.system.SwirldMain;
 import com.swirlds.common.utility.CommonUtils;
 
 /**
  * Singleton factory for loading and registering {@link FCQueue} statistics. This is the primary entry point for all
- * {@link SwirldMain} implementations that wish to track {@link FCQueue} statistics.
+ * SwirldMain implementations that wish to track {@link FCQueue} statistics.
```

Comment:
should be {@link com.swirlds.platform.system.SwirldMain}

Reply:
This link can no longer exist without adding a circular dependency on swirlds common and swirlds core.